client/src/components/AlumniDirectory.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import { Card, Image, Search, Pagination, Grid, Segment } from 'semantic-ui-react'
+
+/*
+props:
+- entries: list of alumni profiles
+*/
+export default class AlumniDirectory extends Component {
+    state = {
+        activePage: 1,
+        numEntries: Object.keys(this.props.entries).length,
+        isLoading: false,
+        value: '',
+        totalPages: Math.ceil(Object.keys(this.props.entries).length/3)
+    }
+
+    handlePaginationChange = (e, { activePage }) => {
+        this.setState({ activePage })
+    }
+    handleSearchChange = (e, { value }) => {
+        this.setState({ value })
+    }
+
+    render(){
+        const {
+            totalPages,
+            activePage
+        } = this.state
+
+        let profiles=[]
+        for (let post in this.props.entries) {
```

Comment:
Might be cleaner to iterate over items using `let post of this.props.entries` and then using each post entry inside iteration like `post.imageUrl`

Reply:
fixed!